client/src/components/infoDrawer/infoFormat.js

```diff
@@ -80,20 +82,47 @@ const renderOrganism = (organism, skeleton) => {
 };
 
 // Render list of metadata attributes found in categorical field
-// Ignores categories with empty or null values
 const renderSingleValueCategories = (singleValueCategories, skeleton) => {
   if (singleValueCategories.size === 0) return null;
   return (
     <>
       <H3 className={skeleton ? Classes.SKELETON : null}>Dataset Metadata</H3>
       <UL>
-        {Array.from(singleValueCategories).map((pair) => {
-          if (!pair[1] || pair[1] === "") return null;
+        {Array.from(singleValueCategories).map((pair, index, array) => {
+          const [category, value] = pair;
+          let ontologyTermID = "";
+          // If the value is empty skip it
+          if (!value || value === "") return null;
+
+          // If the next category is a ontology term, let's save it's value to render now
+          if (array[index + 1]?.[0]?.toString().includes("ontology_term_id"))
+            // ontologyTermID = array[index + 1][1];
+            ontologyTermID = array[index + 1]?.[1];
+          // If this category is an ontology term skip it since we already used it
+          if (category.toString().includes("ontology_term_id")) {
+            return null;
+          }
+
+          const catWidth = "30%";
```

Comment:
seems like we can lift these constants out of the function declaration and put them right above the function?

E.g., 

```tsx
const CAT_WIDTH = "30%";
const VAL_WIDTH = "35%";
const renderSingleValueCategories = (singleValueCategories, skeleton) => { ... }
```